package.json

```diff
@@ -1,16 +1,27 @@
 {
   "name": "gaia-icons",
   "version": "0.8.0",
+  "homepage": "https://github.com/gaia-components/gaia-icons",
+  "license": "MIT",
   "main": "gaia-icons.js",
-  "scripts": {
-    "gh-pages": "git push upstream master:gh-pages"
-  },
   "devDependencies": {
     "grunt": "^0.4.5",
     "grunt-contrib-clean": "^0.5.0",
     "grunt-rename": "^0.1.3",
     "grunt-replace": "^0.7.8",
     "grunt-webfont": "^0.5.1",
+    "jshint": "^2.7.0",
     "optimist": "^0.6.1"
+  },
+  "scripts": {
+    "lint": "./node_modules/jshint/bin/jshint gaia-icons.js",
```

Comment:
`"lint": "jshint gaia-icons.js",`

Should work if `jshint` exists in `node_modules/.bin/`